packages/devtools-reps/src/object-inspector/actions.js

```diff
@@ -72,22 +54,21 @@ function nodeFocus(node: Node) {
  * symbols for a given node. If we do, it will call the appropriate ObjectClient
  * functions.
  */
-function nodeLoadProperties(
-  item: Node,
-  actor?: string,
-  loadedProperties: LoadedProperties,
-  createObjectClient: CreateObjectClient,
-  createLongStringClient: CreateLongStringClient
-) {
-  return async ({ dispatch }: ThunkArg) => {
+function nodeLoadProperties(node: Node, actor) {
+  return async ({ dispatch, client, getState }: ThunkArg) => {
```

Comment:
We have access to `client` because of the debugger's thunkArgs. This was one of the best decisions we made early on in the debugger.  We can easily land this for the console as well.

https://github.com/devtools-html/debugger.html/blob/master/src/utils/bootstrap.js#L48-L50